docassemble/ALAffidavitOfIndigency/data/questions/affidavit_body.yml

```diff
@@ -643,21 +668,23 @@ subquestion: |
 fields:
   - I do not know the docket number: dont_know_docket_number
     datatype: yesno
+    default: ${ "Yes" if case_new_or_existing == "new" else "" }
```

Comment:
I guess this works because of YAML semantics, but could it just be `default: ${ case_new_or_existing == "new" }`?

Reply:
```suggestion
    default: ${ case_new_or_existing == "new" }
```